extract_resources_from_api_tree.py

```diff
@@ -0,0 +1,178 @@
+from logging import log
+import windstream_logger
+
+logging= windstream_logger.get_logger("api_tree_process")
+
+
+import os
+import pandas as pd
+import requests
+from anytree import Node, RenderTree, AsciiStyle, PreOrderIter, LevelOrderIter
+
+# Function to extract the resources from anytree
+def extract_resources_from_api_tree(hb_root, url_returned, token):
```

Comment:
The function is too long and hard to read. Consider extracting logical pieces in sub-functions

Reply:
I agree. I will do that.